source/numerics/matrix_tools.inst.in

```diff
@@ -54,40 +54,40 @@ for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension :  SPACE_DIMENSIONS
        const ConstraintMatrix   &constraints);
 
 
-    template
-      void MatrixCreator::create_mass_matrix<deal_II_dimension,float,deal_II_space_dimension>
-      (const Mapping<deal_II_dimension,deal_II_space_dimension>       &mapping,
-       const DoFHandler<deal_II_dimension,deal_II_space_dimension>    &dof,
-       const Quadrature<deal_II_dimension>    &q,
-       SparseMatrix<float>     &matrix,
-       const Function<deal_II_space_dimension> * const coefficient,
-       const ConstraintMatrix   &constraints);
-    template
-      void MatrixCreator::create_mass_matrix<deal_II_dimension,float,deal_II_space_dimension>
-      (const DoFHandler<deal_II_dimension,deal_II_space_dimension>    &dof,
-       const Quadrature<deal_II_dimension>    &q,
-       SparseMatrix<float>     &matrix,
-       const Function<deal_II_space_dimension> * const coefficient,
-       const ConstraintMatrix   &constraints);
-    template
-      void MatrixCreator::create_mass_matrix<deal_II_dimension,float,deal_II_space_dimension>
-      (const Mapping<deal_II_dimension,deal_II_space_dimension>       &mapping,
-       const DoFHandler<deal_II_dimension,deal_II_space_dimension>    &dof,
-       const Quadrature<deal_II_dimension>    &q,
-       SparseMatrix<float>     &matrix,
-       const Function<deal_II_space_dimension>      &rhs,
-       Vector<double>           &rhs_vector,
-       const Function<deal_II_space_dimension> * const coefficient,
-       const ConstraintMatrix   &constraints);
-    template
-      void MatrixCreator::create_mass_matrix<deal_II_dimension,float,deal_II_space_dimension>
-      (const DoFHandler<deal_II_dimension,deal_II_space_dimension>    &dof,
-       const Quadrature<deal_II_dimension>    &q,
-       SparseMatrix<float>     &matrix,
-       const Function<deal_II_space_dimension>      &rhs,
-       Vector<double>           &rhs_vector,
-       const Function<deal_II_space_dimension> * const coefficient,
-       const ConstraintMatrix   &constraints);
+    //template
+    //  void MatrixCreator::create_mass_matrix<deal_II_dimension,float,deal_II_space_dimension>
```

Comment:
Nah, let's force people to using matching scalar types for the local and global matrices.